src/components/Snippet.jsx

```diff
@@ -18,6 +18,9 @@ class Snippet extends React.Component {
     this.state = { isShowEmbed: false };
     this.toggleEmbed = this.toggleEmbed.bind(this);
     this.download = this.download.bind(this);
+    this.copyClipboard = (e) => {
+      copyToClipboard(e, 'embedded');
```

Comment:
Why it's here? It must be a separate commit! Please, do not make different changes in a single commit.